docs/site/index/downloads.md

```diff
@@ -213,13 +213,23 @@ The 1.0 version of CLDR is described here for historical interest only. It was h
 [main]: https://github.com/unicode-org/cldr.git
 [dev-delta-dtd]: https://www.unicode.org/cldr/charts/dev/supplemental/dtd_deltas.html
 
```

Comment:
Don't we also want to add the corresponding line in the table at the top, i.e.
```
|  49 | 2026-03-XX | [v49][] | [CLDR49][] | [Charts49][] | [LDML49][] | [Δ49][] | [release-49][] | [ΔDtd49][] | [49.0.0][] |
```
?